ckanext/canada/schemas/organization.yaml

```diff
@@ -0,0 +1,145 @@
+scheming_version: 2
+organization_type: organization
+
+about: 'open.canada.ca: Organization metadata definition'
+about_url: http://github.com/open-data/ckanext-canada
+
+form_languages:
+- en
+- fr
+
+#
+#   ORGANIZATION FIELDS
+#
+fields:
+
+# Field = Title.
+# {The organization's title}
+- field_name: title_translated
+  label:
+    en: Title
+    fr: Titre
+  fluent_form_label:
+    en:
+      en: "Title (English)"
+      fr: "Titre (anglais)"
+    fr:
+      en: "Title (French)"
+      fr: "Titre (français)"
+  fluent_help_text:
+    en:
+      en: The English name by which the organization is known
+      fr: pending translation
+    fr:
+      en: The French name by which the organization is known
+      fr: pending translation
+  required: true
+  # copied from fluent_text preset
+  form_snippet: fluent_text.html
+  display_snippet: fluent_text.html
+  error_snippet: fluent_text.html
+  validators: fluent_text unicode scheming_required
+  output_validators: fluent_core_translated_output
+
+# Field = Description.
+# {The organization's description}
+- field_name: notes_translated
+  label:
+    en: Description
+    fr: Description
+  fluent_form_label:
+    en:
+      en: "Description (English)"
+      fr: "Description (anglais)"
+    fr:
+      en: "Description (French)"
+      fr: "Description (français)"
+  fluent_help_text:
+    en:
+      en: A little information about the organization in English
+      fr: pending translation
+    fr:
+      en: A little information about the organization in English
+      fr: pending translation
+  # copied from fluent_markdown preset
+  form_snippet: fluent_markdown.html
+  display_snippet: fluent_markdown.html
+  error_snippet: fluent_text.html
+  validators: fluent_text
+  output_validators: fluent_core_translated_output
+
+# Field = Short Form.
+# {The organization's short form}
+- field_name: shortform_translated
```

Comment:
this field can be called `shortform`. the `_translated` field names were added as a way to add fluent fields that override core fields, and there's no core field called `shortform`